### PR TITLE
backport: Prometheus for the app

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
Combined backport of:
1. https://github.com/FlowFuse/flowfuse/pull/2889
2. https://github.com/FlowFuse/flowfuse/pull/2893

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

### CODE REVIEW

File: forge/forge.js [hunk 1]
```diff
@@ -75,6 +75,20 @@ module.exports = async (options = {}) => {
...
+        await server.register(metricsPlugin, { endpoint: '/metrics' })
+    }
+
+    if (runtimeConfig.telemetry.backend?.sentry?.dsn) {
+        server.register(require('@immobiliarelabs/fastify-sentry'), {
+            dsn: runtimeConfig.telemetry.backend.sentry.dsn,
+            environment: process.env.NODE_ENV,
+            release: `flowforge@${runtimeConfig.version}`
+        })
+    }
```
Comment (hardillb) [inline forge/forge.js:90]: Did you mean to include the `sentry` stuff in here? That has more changes than just this and the dependency is not in the `package.json` included here.

```suggestion
```

**Review by hardillb — COMMENTED** [review]
Need to check the sentry inclusion.

File: forge/forge.js (hunk 1 above)
Comment (knolleary) [reply forge/forge.js:90]: We are validating the sentry work in staging before deciding whether to adopt in production as well (which is why it didn't get backported automatically).

Given this is guarded on `runtimeConfig.telemetry.backend?.sentry?.dsn` - which should not get set in production then this should be okay as is.

Comment (knolleary) [reply forge/forge.js:90]: Ah - but the linting won't like the `require` if not also in package.json.

Let me discuss with @Pezmc if there's any reasons at this point not to get sentry setup on production as well.

Comment (ZJvandeWeg) [reply forge/forge.js:90]: @hardillb Didn't spot this in the diff, usually cherry-pick doesn't show one. As the sentry changes created a conflict, I figured it was already in the maintenance branch.

I can manually remove it for now, to make it easier for this PR to go through without any additional scope inflation.

**Comment by hardillb** [comment]
Going to close this given Nicks announcement

**Pull request closed without merging**